azure-pipelines.yml

```diff
@@ -55,14 +55,14 @@ jobs:
     inputs:
       command: test
       projects: '**/*UnitTest/*.csproj'
-      arguments: '--configuration $(buildConfiguration) /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura'
```

Comment:
Removing due to errors related to https://github.com/tonerdo/coverlet/issues/210

Reply:
Will implement necessary changes in #138